.fixtures.yml

```diff
@@ -4,5 +4,6 @@ fixtures:
     extlib:        "git://github.com/puppet-community/puppet-extlib.git"
     foreman:       "git://github.com/theforeman/puppet-foreman.git"
     common:        "git://github.com/katello/puppet-common.git"
+    evenup:        "git://github.com/evenup/evenup-trusted_ca.git"
```

Comment:
Ideally the name here would be the name of the puppet module and not the provider of the module, i.e. trusted_ca and not evenup